src/style/content/preferences/options.less

```diff
@@ -52,6 +52,12 @@
     }
   }
 
+  input[type='radio']:focus-visible + label {
+    border-radius: 4px;
+    outline: 1px solid var(--accent-color);
+    outline-offset: 0.4rem;
+  }
+
```

Comment:
Forgot to use the new classes here? Or did you intend to use `var(-accent-color)` instead of `var(-accent-color-focus)` ?

Reply:
yes. updated.